scripts/im_plot.sl

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+# script version: slurm
+#
+# must be run with sbatch clean_up.sl [realisation directory] [realisation name] [management database location]
+
+#SBATCH --job-name=im_plot
+#SBATCH --account=nesi00213
+#SBATCH --partition=prepost
+#SBATCH --time=00:30:00
+#SBATCH --cpus-per-task=4
+
+if [[ ! -z ${CUR_ENV} && ${CUR_HPC} != "mahuika" ]]; then
+    source $CUR_ENV/workflow/install_workflow/helper_functions/activate_env.sh $CUR_ENV "mahuika"
+fi
+
+
+CSV_PATH=$1
+RRUP_OR_STATION_PATH=$2
+OUTPUT_XYZ_DIR=$3
+
+SRF_PATH=$4
+MODEL_PARAMS=$5
+
+MGMT_DB_LOC=$6
+SRF_NAME=$7
+
+script_start=`date`
+echo "script started running at: $script_start"
+runtime_fmt="%Y-%m-%d_%H:%M:%S"
+timestamp=`date +%Y%m%d_%H%M%S`
+
+start_time=`date +${runtime_fmt}`
+echo ___im plot___
+
+python $gmsim/workflow/scripts/cybershake/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $SRF_NAME IM_plot running
+res=`python $gmsim/visualization/im_plotting/im_plot.py $CSV_PATH $RRUP_OR_STATION_PATH --output $OUTPUT_XYZ_DIR`
+
+exit_val=$?
+
+if [[ $exit_val == 0 ]]; then
+
+    echo "finished making csvs for im_plot"
+    ## Load python2 for plot_stations
+    module load Python/2.7.14-gimkl-2017a
```

Comment:
Should be using the load_python2_mahuika command rather than the module change. Changes to the PYTHONPATH inside/outside of environments do weird things.

Reply:
@claudio525 When you get back can you have a look at this? Just to understand why we can't call this function from a sl script?